site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1486,48 +1470,6 @@ public function getLateDayCacheForUser(string $user_id): array {
         return $late_day_events;
     }
 
-    /**
-     * Get the late day information for a specific user (graded gradeable information)
-     * @param string $user_id
-     * @param string $g_id
-     * @return null|array $return = [
-     *      'g_id' => string,
-     *      'user_id' => string,
-     *      'team_id' => string,
-     *      'late_days_allowed' => int,
-     *      'late_day_date' => DateTime,
-     *      'submission_days_late' => int,
-     *      'late_day_exceptions' => int,
-     *      'late_days_remaining' => int,
-     *      'late_day_status' => int,
-     *      'late_days_change' => int
-     * ]
-     */
-    public function getLateDayCacheForUserGradeable(string $user_id, string $g_id): ?array {
-        $params = [$user_id, $g_id];
-        $query = "SELECT * FROM late_day_cache
-                    WHERE user_id=?
-                    AND g_id=?";
-        $this->course_db->query($query, $params);
-
-        $row = $this->course_db->row();
-
-        // If cache doesn't exist, generate it and query again
-        if (empty($row)) {
-            $this->generateLateDayCacheForUser($user_id);
-            $this->course_db->query($query, $params);
-            $row = $this->course_db->row();
-        }
-
-        // If cache still doesn't exist, the gradeable is not associated with
-        // LateDays OR there has been a computation error
-        if (empty($row)) {
-            return null;
-        }
-
-        return $row;
-    }
-
```

Comment:
This isn't currently used but it looks like it was planned on being used in https://github.com/Submitty/Submitty/pull/7424.